### PR TITLE
[Misc] Update `w4a16` `compressed-tensors` support to include `w8a16`

### PR DESCRIPTION
**Pull request by dsikka**
# Summary
- Expand support such that we can run `w8a16` models compressed through [`compressed-tensors`](https://github.com/neuralmagic/compressed-tensors)
- This requires changing the supported scheme from `CompressedTensorsW4A16` to `CompressedTensorsWNA16` to support both w4a16 and w8a16/allow use of the same kernels

By Neural Magic

### CODE REVIEW

**Comment by robertgshaw2-neuralmagic** [comment]
LGTM - pending test failure which relied on the old name

**Comment by dsikka** [comment]
@robertgshaw2-neuralmagic updated

File: tests/quantization/test_compressed_tensors.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def test_compressed_tensors_w4a16(vllm_runner, w4a16_args):
 
         qkv_proj = layer.self_attn.qkv_proj
         assert isinstance(qkv_proj.quant_method, CompressedTensorsLinearMethod)
-        assert isinstance(qkv_proj.scheme, CompressedTensorsW4A16)
+        assert isinstance(qkv_proj.scheme, CompressedTensorsWNA16)
```
Comment (robertgshaw2-neuralmagic) [inline tests/quantization/test_compressed_tensors.py:89]: Can you add a test case for w8a16?

**Review by robertgshaw2-redhat — CHANGES_REQUESTED** [review]
Please add a test case. Otherwise LGTM

**Review by robertgshaw2-redhat — APPROVED** [review]
lgtm

**Pull request merged**